cvat/apps/engine/ddln/tasks/__init__.py

```diff
@@ -0,0 +1,23 @@
+from .spotter import SpotterTaskHandler
+from .vls import VlsTaskHandler
+
+
+def create_task_handler(task_type):
+    if task_type == "spotter":
+        return SpotterTaskHandler()
+    if task_type == "vls":
+        return VlsTaskHandler()
+    raise ValueError("Unexpected task type: {!r}".format(task_type))
+
+
+def guess_task_type(task):
```

Comment:
why is this necessary?

Reply:
Task type is identified by the labels which were added to the task. it should almost always properly detect task type, but it might fail if wrong labels were attached on task creation.

Identified task type is then used to run right logic on task validation or task export.